pkg/oc/admin/policy/review.go

```diff
@@ -85,7 +85,7 @@ func NewCmdSccReview(name, fullName string, f *clientcmd.Factory, out io.Writer)
 
 func (o *sccReviewOptions) Complete(f *clientcmd.Factory, args []string, cmd *cobra.Command, out io.Writer) error {
 	if len(args) == 0 && len(o.FilenameOptions.Filenames) == 0 {
-		return kcmdutil.UsageErrorf(cmd, cmd.Use)
+		return kcmdutil.UsageErrorf(cmd, "one or more resources must be specified with -f or as <resource> <name> or <resource>/<name>")
```

Comment:
Make it short: `one or more resources must be specified`, it's not worth getting into details. One will be presented with information about `-h` where more information is available.